sensu-plugins-signifai.gemspec

```diff
@@ -9,7 +9,10 @@ Gem::Specification.new do |s|
 
   s.date                   = Date.today.to_s
   s.description            = 'This plugin provides a Sensu handler for SignifAi'
-  s.email                  = '<zcarlson@signifai.io>'
+  s.email                  = [
+    '<sensu-users@googlegroups.com>',
+    '<zcarlson@signifai.io>'
```

Comment:
given the scope of the PR I think that should be a different submission. I will create an issue for it.